backend/src/news/schemas.py

```diff
@@ -4,4 +4,8 @@ class PromptRequest(BaseModel):
     prompt: str
 
 class NewsSumaryRequestSchema(BaseModel):
-    content: str
\ No newline at end of file
+    content: str
+
+class NewsSumaryCustomModelSchema(BaseModel):
+    content: str
+    ai_model: str 
```

Comment:
Since `openai` and `anthropic` are the only 2 available options for the field `llm_model`. You may consider use `Literal[...]` or `StrEnum` instead of simply `str` to help verify the received HTTP requests and simplify the conditional statements.

```suggestion
class NewsSummaryCustomModelRequestSchema(BaseModel):
    content: str
    ai_model: Literal["openai", "anthropic"]
```